configs/srk.yaml

```diff
@@ -0,0 +1,52 @@
+# This provider will be used by srk commands unless specifically overwritten
```

Comment:
make a template file for now, let people copy it over and modify the fields that they need.